### ISSUE TITLE
Button color only on bottom row

### ISSUE THREAD

**Issue by eavyon** [Blackymas/NSPanel_HA_Blueprint#1356]
Just a minor design issue. It looks like only the bottom row of the buttons get a color in the on state. Upper row stays always gray.

Could probably pick this up when looking into this: https://github.com/Blackymas/NSPanel_HA_Blueprint/issues/1184

**Comment by edwardtfn** [comment]
Could you please send a picture so I can understand what you are talking about?

Also, which versions of TFT, ESPHome and Blueprint are shown on the blue boot page?

**Comment by eavyon** [comment]
Only bottom row icons do have a color.
![IMG_20231128_231051](https://github.com/Blackymas/NSPanel_HA_Blueprint/assets/84970882/0f0aa51f-7d3c-49d5-be51-9234adf292b7)

**Comment by eavyon** [comment]
You are right about the generic page for binary and rgb lights but other domains works a bit different? I shuffled the buttons around and there is no difference between bottom and upper row but its dependant on the entity.
I tried a binary light and color stays gray. I tried a switch and color does change. A cover does also change color.

**Comment by edwardtfn** [comment]
Ok, thanks for the additional info.
The system does behave differently based on the domain, so it's probably broken in the light branch.
I will take a look at that.

**Comment by edwardtfn** [comment]
On the blue boot screen, which versions you see for TFT, ESPHome and Blueprint?

**Comment by eavyon** [comment]
TFT 4.1.1
ESPHome 4.1.3
Blueprint 4.1.1

**Comment by edwardtfn** [comment]
Please update your blueprint to v4.1.3.
There is where the fix is.
